### PR TITLE
refactor: dockerfile optimized for smaller build

### PR DESCRIPTION
**Pull request by jim80net**
### Pre requirements:
- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/lightdash/lightdash/blob/main/.github/CONTRIBUTING.md#sending-a-pull-request).

### Reference:
Continuation of https://github.com/lightdash/lightdash/pull/1361 and https://github.com/lightdash/lightdash/pull/1370

### Description:
Adds python into the runtime OS. 

### Preview:
Fixes this issue: where /usr/bin/entrypoint.sh (from jaffle fails to execute). 

image
### Scope of the changes (select all that apply):
- [ ] Frontend  
- [ ] Backend  
- [ ] Common  
- [ ] E2E tests  
- [ ] Docs  
- [X] CI/CD

### CODE REVIEW

File: dockerfile [hunk 1]
```diff
@@ -1,69 +1,55 @@
...
+    libsasl2-modules-gssapi-mit \
+    nodejs \
+    python3 \
+    python3-psycopg2 \
+    python3-venv \
+    software-properties-common \
     unixodbc-dev \
-    software-properties-common
-
-# Install latest git
```
Comment (owlas) [inline dockerfile:36]: @jim80net I merged this yesterday but dropped `git` from the container, do we need to add this to the `apt` package list?

**Review by owlas — CHANGES_REQUESTED** [review]

File: dockerfile (hunk 1 above)
Comment (jim80net) [reply dockerfile:36]: For git, no. git is available by default in node:14-bullseye

image

Comment (owlas) [reply dockerfile:36]: That's so bizarre, I'll test out now

Comment (jim80net) [reply dockerfile:36]: I think I found what was missing: 65dd8fd: I was using `node:14-slim` when I thought it was `node:14-bullseye`.

**Pull request merged**